buildSrc/src/main/groovy/com/microsoft/applicationinsights/build/tasks/PropertiesFileGenerator.groovy

```diff
@@ -43,8 +44,9 @@ class PropsFileGen extends DefaultTask {
         }
 
         Properties oldProps = new Properties();
-        FileInputStream inFileStream = new FileInputStream(targetFile);
-        oldProps.load(inFileStream);
+        targetFile.withInputStream { inFileStream ->
```

Comment:
do this in a groovy way, and also, close the stream so the build doesn't hang :)